src/pages/safety-&-side-effects.astro

```diff
@@ -1,18 +1,270 @@
+
+          <ul class="side-effects-page--list">
+            <li>
+              loss of vision
+            </li>
+            <li>
+              dark spots in your child’s vision (floaters)
+            </li>
+            <li>
+              other changes to your child’s vision
+            </li>
+          </ul>
+
+          <p class="side-effects-page--section__paragraph"><strong>Severe diarrhea:</strong> Diarrhea is common with Koselugo and can also be severe. Tell your child’s healthcare provider right away the first time that your child gets diarrhea during treatment with Koselugo. Your child’s healthcare provider may give your child medicine to help control diarrhea and may tell them to drink more fluids</p>
+
+          <p class="side-effects-page--section__paragraph"><strong>Skin rash:</strong> Skin rashes are common with Koselugo and can also be severe. Tell your child’s healthcare provider if your child gets any of the following signs or symptoms:</p>
+
+          <ul class="side-effects-page--list">
+            <li>rash that covers a large area of your child’s body</li>
+            <li>blisters</li>
+            <li>peeling skin</li>
+          </ul>
+
+          <p class="side-effects-page--section__paragraph"><strong>Muscle problems (rhabdomyolysis):</strong> Muscle problems are common with Koselugo and can also be severe. Treatment with Koselugo may increase the level of a muscle enzyme in your child’s blood, which may be a sign of muscle damage. Your child’s healthcare provider should do a blood test to check your child’s muscle enzyme levels before they start taking Koselugo and during treatment. Tell your child’s healthcare provider right away if your child gets any of the following signs or symptoms:</p>
+
+          <ul class="side-effects-page--list">
+            <li>muscle aches or pain</li>
+            <li>dark, reddish urine</li>
+            <li>muscle spasms and weakness</li>
+          </ul>
+
+          <h3 class="side-effects-page--section__heading side-effects-page--section__heading__3">
+            Your child’s healthcare provider may change your child’s dose, temporarily stop, or permanently ask your child to stop taking Koselugo if they have any of these side effects.
+          </h3>
+          <br>
+
+          <h3 class="side-effects-page--section__heading side-effects-page--section__heading__3">The most common side effects of Koselugo are:</h3>
+
+
+          <div class="side-effects-page--commun-list">
+            
+            <div class="side-effects-page--commun-list__column">
+              
+              <div class="side-effects-page--commun-list__item">
+                <img class="side-effects-page--commun-list__item__img" src="/assets/images/safety-and-side-effects/icons/vomiting-icon.png" alt=" " srcset="">
```

Comment:
![image](https://user-images.githubusercontent.com/31933833/228646082-46811368-1448-4a76-a75a-8200a861a17f.png)

Dots appear too big

This es figma:
![image](https://user-images.githubusercontent.com/31933833/228646274-1611a1ab-c6f2-4492-abd6-f9b6c635710c.png)